### PR TITLE
[editor] Fix exportStart option and add zeroFilledMemory option

### PR DESCRIPTION
**Pull request by MaxGraey**

### CODE REVIEW

File: src/.vuepress/public/editor.html [hunk 1]
```diff
@@ -780,14 +788,17 @@ <h5>Documentation</h5>
...
         if (optionExportTable.checked) {
           options.push('--exportTable')
         }
         if (optionImportTable.checked) {
           options.push('--importTable')
         }
-        if (optionExplicitStart.checked) {
-          options.push('--explicitStart')
+        if (optionExportStart.checked) {
+          options.push('--exportStart')
```
Comment (dcodeIO) [inline src/.vuepress/public/editor.html]: I believe `--exportStart` takes a string argument (typically either `_start` or `_initialize`), or is it working this way?

Comment (MaxGraey) [reply src/.vuepress/public/editor.html]: Ah right! Will improve this

Comment (MaxGraey) [reply src/.vuepress/public/editor.html]: I guess better just remove this option completely

Comment (dcodeIO) [reply src/.vuepress/public/editor.html]: Wdyt of keeping the option and adding a text field after, like `[ ] Export start [_start]` where `_start` is the default? Could imagine that it's useful for some use cases.

Comment (MaxGraey) [reply src/.vuepress/public/editor.html]: I thought so too at first, but we can easily do this just by adding `import "wasi"` [see example in playground](https://www.assemblyscript.org/editor.html#IyFvcHRpbWl6ZT1zcGVlZCZydW50aW1lPXN0dWIKaW1wb3J0ICJ3YXNpIgojIWh0bWwKPHRleHRhcmVhIGlkPSJvdXRwdXQiIHN0eWxlPSJoZWlnaHQ6IDEwMCU7IHdpZHRoOiAxMDAlIiByZWFkb25seT48L3RleHRhcmVhPgo8c2NyaXB0IHR5cGU9Im1vZHVsZSI+CmNvbnN0IGV4cG9ydHMgPSBhd2FpdCBpbnN0YW50aWF0ZShhd2FpdCBjb21waWxlKCksIHsgLyogaW1wb3J0cyAqLyB9KQpjb25zdCBvdXRwdXQgPSBkb2N1bWVudC5nZXRFbGVtZW50QnlJZCgnb3V0cHV0JykKZm9yIChsZXQgaSA9IDA7IGkgPD0gMTA7ICsraSkgewogIG91dHB1dC52YWx1ZSArPSBgZmliKCR7aX0pID0gJHtleHBvcnRzLmZpYihpKX1cbmAKfQo8L3NjcmlwdD4=)

Comment (dcodeIO) [reply src/.vuepress/public/editor.html]: It's a side-effect of `import "wasi"`, yeah, but doing so will also reconfigure the module, with another `abort` etc.

Comment (MaxGraey) [reply src/.vuepress/public/editor.html]: Ok, done!

File: src/.vuepress/public/editor.html [hunk 2]
```diff
@@ -253,9 +253,10 @@ <h5>Features</h5>
         <div class="option">
           <p><input id="option-noExportMemory" type="checkbox" /><label for="option-noExportMemory"> Don't export memory to host</label></p>
           <p><input id="option-importMemory" type="checkbox" /><label for="option-importMemory"> Import memory from host</label></p>
+          <p><input id="option-zeroFilledMemory" type="checkbox" disabled/><label for="option-zeroFilledMemory"> Assume imported memory is zeroed</label></p>
           <p><input id="option-exportTable" type="checkbox" /><label for="option-exportTable"> Export function table to host</label></p>
           <p><input id="option-importTable" type="checkbox" /><label for="option-importTable"> Import function table from host</label></p>
-          <p><input id="option-explicitStart" type="checkbox" /><label for="option-explicitStart"> Explicit start function</label></p>
+          <p><input id="option-exportStart" type="checkbox" /><label for="option-exportStart"> Explicit export start as _start</label></p>
```
Comment (dcodeIO) [inline src/.vuepress/public/editor.html]: Perhaps
```suggestion
          <p><input id="option-exportStart" type="checkbox" /><label for="option-exportStart"> Export start function (as _start)</label></p>
```

**Review by dcodeIO — APPROVED** [review]

**Pull request merged**